infra.bs

```diff
@@ -167,8 +167,8 @@ over.
 </div>
 
 <p class=example id=example-variable-ternary>Let <var ignore>activationTarget</var> be
-<var ignore>target</var>, if <var ignore>isActivationEvent</var> is true and target has activation
-behavior, and null otherwise.
+<var ignore>target</var> if <var ignore>isActivationEvent</var> is true and target has activation
+behavior, otherwise null.
```

Comment:
As @sideshowbarker noted this should be a semicolon, not a comma.

Reply:
That was in reaction to the ", null otherwise" variant. Also, it's not the style we used in https://infra.spec.whatwg.org/#code-unit-prefix.